src/main/java/juja/sqlcmd/Application.java

```diff
@@ -1,7 +1,101 @@
 package juja.sqlcmd;
 
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
 public class Application {
-    public static void main(String[] args) {
+    private static final String DB_DRIVER = "org.postgresql.Driver";
+    private static final String DB_CONNECTION_URL = "jdbc:postgresql://localhost:5432/sqlcmd";
+    private static final String DB_USER = "sqlcmd";
+    private static final String DB_PASSWORD = "sqlcmd";
+
+    private Connection connection;
+
+    public static void main(String[] args) throws SQLException, ClassNotFoundException {
+        new Application().simpleSQL();
+    }
+
+    public void simpleSQL() throws SQLException, ClassNotFoundException {
+        Class.forName(DB_DRIVER);
+        connection = DriverManager
+                .getConnection(DB_CONNECTION_URL, DB_USER, DB_PASSWORD);
+        dropTableIfExists("\"user\"");
+        printTableNames();
+        String sqlQuery = "CREATE TABLE \"user\"(id SERIAL PRIMARY KEY, name TEXT, password TEXT)";
+        executeSqlQuery(sqlQuery);
+        printTableNames();
+        insertUser("user1", "password1");
+        insertUser("user2", "password2");
+        insertUser("user3", "password3");
+        printTable("\"user\"");
+        updateUser("name", "user2", "userсhange1");
+        printTable("\"user\"");
+        deleteUser("name", "user3");
+        printTable("\"user\"");
+        connection.close();
+    }
+
+    private void dropTableIfExists(String tableName) {
+        String sqlQuery = "DROP TABLE " + tableName;
```

Comment:
It might be better to use a query like this -> "DROP TABLE IF EXISTS" + tabletsName
[PostgreSQL drop table](https://www.postgresql.org/docs/8.2/static/sql-droptable.html)
What do you think?
You do not process the exception.